### DISCUSSION TITLE
Database Schema using SQL Migration files

### DISCUSSION THREAD

**Discussion by JMEspiz** [rust-lang-ve/hilow#20] [General]
Hey Guys, how are you doing?

I would like to ask you,  how are we going to define and handle updating over database schema?
If we are going to use the migration files strategy, which lib would you suggest?

I was checking the DB connection out, we are using [sqlx lib](https://crates.io/crates/sqlx), but reading the docs I didn't see any reference about migration files.

I would like to take this task.

greetings

**Comment by morenol** [comment]
Hi @JMEspiz, the migrations are done using the [sqlx-cli tool](https://crates.io/crates/sqlx-cli).

**Reply by JMEspiz** [reply]
@morenol awesome, thanks!! I'm going to add the initial migrations then.

**Reply by LeoBorai** [reply]
I've been working on signup handler locally, I will create a draft PR with the progress I already have. I think you could start from there perhaps?

**Reply by LeoBorai** [reply]
Please check on this PR, https://github.com/rust-lang-ve/hilow/pull/21 perhaps you could rebase your local version with that?

**Comment by LeoBorai** [comment]
@JMEspiz , @morenol I opened this PR: https://github.com/rust-lang-ve/hilow/pull/21 ASAP, in order to have @JMEspiz join us in this journey!

I've added comments on the PR because theres clearly stuff halfway there, but this PR implements the basic scaffold we want to have and some logic to improve, but most importantly opens the way for others to contribute
